lib/bencher_adapter/src/results/adapter_results.rs

```diff
@@ -65,6 +96,35 @@ pub enum IaiCallgrindMeasure {
     AtTEndBlocks(JsonNewMetric),
     ReadsBytes(JsonNewMetric),
     WritesBytes(JsonNewMetric),
+
+    /*
+     * Memcheck tool:
+     */
+    MemcheckErrors(JsonNewMetric),
+    MemcheckContexts(JsonNewMetric),
+    MemcheckSuppressedErrors(JsonNewMetric),
+    MemcheckSuppressedContexts(JsonNewMetric),
+
+    /*
+     * Helgrind tool:
+     */
+    HelgrindErrors(JsonNewMetric),
+    HelgrindContexts(JsonNewMetric),
+    HelgrindSuppressedErrors(JsonNewMetric),
+    HelgrindSuppressedContexts(JsonNewMetric),
+
+    /*
+     * Drd tool:
+     */
+    DrdErrors(JsonNewMetric),
+    DrdContexts(JsonNewMetric),
+    DrdSuppressedErrors(JsonNewMetric),
+    DrdSuppressedContexts(JsonNewMetric),
+
+    /*
+     * Unknown
+     */
+    Unknown(JsonNewMetric),
```

Comment:
I really like the catch all here! Nice!

Can we make this a unit variant instead?

```suggestion
    Unknown,
```

From what I can tell, we are never using the tuple value. This also gets rid of the `Unknown` measure type definition. Maybe I'm missing something though?